src/Language/Futhark/TypeChecker/Monad.hs

```diff
@@ -97,8 +97,8 @@ data TypeError = TypeError SrcLoc Notes Doc
 
 instance Pretty TypeError where
   ppr (TypeError loc notes msg) =
-    "Error at" <+> text (locStr loc) <> ":" </>
-    msg <> ppr notes
+    text (inRed "Error") <> ":" <+> msg </>
+    "  at:" <+> text (locStr loc) <+> ppr notes
```

Comment:
I don't think this works grammatically for many of the existing error messages.  Could you maintain the current phrasing and just put `Error` in red?  I'm OK with discussing whether the overall format for errors should change, but it seems like a distinct thing from introducing colour.

Reply:
Ok, sounds good! Better to rephrase later.